sdks/python/apache_beam/yaml/json_utils.py

```diff
@@ -267,3 +279,52 @@ def json_formater(
   convert = row_to_json(
       schema_pb2.FieldType(row_type=schema_pb2.RowType(schema=beam_schema)))
   return lambda row: json.dumps(convert(row), sort_keys=True).encode('utf-8')
+
+
+def _validate_compatible(weak_schema, strong_schema):
+  if not weak_schema:
+    return
```

Comment:
Similar to my other comment, but wouldn't this pass silently? I'm not sure how it would reach this state in the first place, but if the incoming PCollection does not have a schema (perhaps if preceded by a transform that does not output Row?), this would pass validation even if given a json schema to validate against.

Reply:
Again, weak_schema could be `{}`. This comes up in practice if we don't know anything about this part of the input (e.g. it's `Any`), which is also the fallback for `beam_type_to_json_type`.